quipucords/fingerprinter/tests_fingerprint_task.py

```diff
@@ -410,7 +410,7 @@ def _validate_vcenter_result(self, fingerprint, fact):
         self.assertEqual(fact.get('vm.dns_name'),
                          fingerprint.get('vm_dns_name'))
         self.assertEqual(fact.get('vm.host.name'),
-                         fingerprint.get('vm_host'))
+                         fingerprint.get('virtual_host_name'))
 
         self.assertEqual(fact.get('vm.host.cpu_count'),
```

Comment:
Do we need to test for the `virtual_host_name` fact here?  Am I right in thinking that the `vm.host.name` & `virtual_host_name` are separate facts?

Reply:
The names in details don't always match the deployments.